src/sentry/runner/initializer.py

```diff
@@ -47,6 +49,18 @@ def init_plugin(plugin):
         for cls in plugin.get_custom_contexts() or ():
             contexttype(cls)
 
+    if (hasattr(plugin, 'get_celerybeat_schedule') and plugin.is_enabled()):
```

Comment:
FYI, I'd like to get off of the naming of `celery` since at some point in the future, we'll likely be switched away from using `celery`, so I'd rather not dig ourselves deeper into this naming. We can name these seomthing like, `get_cron_schedule()`, and `get_worker_imports()`, `get_worker_queues()`.